core/transport/src/main/java/company/transport/models/Cargo.java

```diff
@@ -0,0 +1,27 @@
+package company.transport.models;
+
+import ch.qos.logback.core.net.server.Client;
+
+import java.time.LocalTime;
+
+public class Cargo {
+    public String Id;
+    public String CompanyId;
+    public Client[] Clients;
+    public String Destination;
+    public LocalTime Departure;
+    public LocalTime Arrival;
+    public Double Weigth;
+    public Double Price;
+
+    public Cargo(String id, String companyId, Client[] clients, String destination, LocalTime departure, LocalTime arrival, Double weigth, Double price) {
+        this.Id = id;
+        this.CompanyId = companyId;
```

Comment:
Add a `Company `property as well. So we can access it directly as `Cargo.company`